src/main/java/ch/njol/skript/bukkitutil/BukkitUnsafe.java

```diff
@@ -65,6 +65,11 @@ public class BukkitUnsafe {
 	@Nullable
 	private static Map<Integer,Material> idMappings;
 
+	@Deprecated
+	public static Material getMaterialFromMinecraftId(String id) {
+		return getMaterialFromNamespacedId(id);
+	}
+
 	@Nullable
 	public static Material getMaterialFromNamespacedId(String id) {
```

Comment:
Some javadocs for these methods would be nice while you're here!

Reply:
Maybe like this?